kfs-core/src/main/java/edu/arizona/kfs/sys/batch/service/impl/RoomImportServiceImpl.java

```diff
@@ -213,124 +213,103 @@ private void updateRoom(String buildingCode, String campusCode, Room matchingRoo
 		else {
 			kfsRoomValue = matchingRoom.getBuildingRoomDescription();
 		}
-		if (!archRoomValue.equals(kfsRoomValue)) {
-			reportMsg += "BuilsingRoomDescription(" + matchingRoom.getBuildingRoomDescription() + "), ";
+		if (!archRoomValue.equalsIgnoreCase(kfsRoomValue)) {
+			reportMsg += "BuildingRoomDescription(" + matchingRoom.getBuildingRoomDescription() + "), ";
 			matchingRoom.setBuildingRoomDescription(archibusRoom.getBuildingRoomDescription());
 		}
-		
-		if (!matchingRoom.isActive()) {
+		if(!matchingRoom.isActive()){
 			reportMsg += "Active(false)";
 			matchingRoom.setActive(true);
 		}
 		
-		matchingRoom.setActive(true);
```

Comment:
This line was removed because if statement above is already setting matchingRoom.setActive to true.